### DISCUSSION TITLE
Dịch outlier

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#44] [Terminology] [answered]
Từ này mọi người thường dịch là `ngoại lai` rồi nhưng chưa được đưa vào glossary.

@mlbvn/handson-ml2

**Comment by tiepvupsu** [comment]
`ngoại lệ` thì sao nhỉ?

**Comment by rootonchair** [comment, marked as answer]
Trong ngữ cảnh thống kê thì từ `ngoại lai` được dùng phổ biến hơn. Còn từ `ngoại lệ` có thể được dùng trong một số trường hợp thông thường. Em thấy mình nên thêm vào bảng nhưng có thể chú thích thêm context là thống kê
